### ISSUE TITLE
Move classifier training from notebook to script

### ISSUE THREAD

**Issue by wbwakeman** [AllenInstitute/DeepCell#18]
Our classifier training currently lives in a notebook.  This is suitable for prototyping but not ideal for production work.  This code should be implemented in scripts that can be run on a backend that is either SageMaker or on-prem.

https://github.com/AllenInstitute/DeepCell/blob/master/scripts/notebooks/train_evalute_2020_extended_classifier.ipynb

Implementing into CLI using argschema.
Create a module to run training 
Add a SageMaker runner (reference [Adam's code for DeepInterpolation](https://github.com/AllenInstitute/ophys_etl_pipelines/pull/434))

All training hyperparameters should be exposed in module.
Includes e.g. learning_rate_decay, dropout, many others
(Adam provide list)

Should design and agree upon:
where/how we store outputs to the model and metrics that we calculate (for SageMaker, this is clear and obvious - it does this for us)
